### DISCUSSION TITLE
Access rule for Configdumps

### DISCUSSION THREAD

**Discussion by ifindthanh** [Keyfactor/ejbca-ce#89] [Q&A] [answered]
Dear guys,

I'm creating an example to use EJBCA Configdump Rest API.
Things are okey when I use the default superadmin.p12 file for client authentication.
The problem is, I want to create an extra role to reduce some administration rules compared to super admin one, it doesn't work. The response of RestAPI always is `The admin is not authorized to import Configdumps!`

I tried to figure it out in the Rule configuration - Advanced mode but cannot find any Role for configdump

Can you please help me!

**Comment by mike-agrenius-kushner** [comment, marked as answer]
Hi,

Sorry about our rules system – it's usually too complex for its own good. 

The rule you need is /system_functionality/edit_systemconfiguration

Cheers,
Mike

**Reply by ifindthanh** [reply]
@mike-agrenius-kushner thank you very much, it works for my side
